### PR TITLE
add working vote menu files

### PR DESCRIPTION
**Pull request by reaaLx**

### CODE REVIEW

**Comment by Dss0** [comment]
how do you open the menu tho? would prolly be good if it could be accessed in a way like the quickmessage menu. Also needs to be added to the control options (keybind) and default_mp_controls.cfg.

**Comment by 4doorsmorehoes** [comment]
imo should be open in the class menu options

**Comment by Dss0** [comment]
nice job!

File: iw4x/iw4x_00/ui_mp/scriptmenus/changegametype.menu [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+				setItemColor self backcolor 0 0 0 1;
+			}
+			leaveFocus
+			{
+				setItemColor self backcolor 0 0 0 0;
+			}
+			action
+			{
+				close "self"; 
+                uiScript "voteTypeMap";
```
Comment (simonlfc) [inline iw4x/iw4x_00/ui_mp/scriptmenus/changegametype.menu:121]: uiScript "voteGame";

File: iw4x/iw4x_00/ui_mp/scriptmenus/changemap.menu [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+				setItemColor self backcolor 0 0 0 1;
+			}
+			leaveFocus
+			{
+				setItemColor self backcolor 0 0 0 0;
+			}
+			action
+			{
+				close "self"; 
+                uiScript "voteTypeMap";
```
Comment (simonlfc) [inline iw4x/iw4x_00/ui_mp/scriptmenus/changemap.menu:123]: uiScript "voteMap";

File: iw4x/iw4x_00/ui_mp/scriptmenus/kickplayer.menu [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+				setItemColor self backcolor 0 0 0 1;
+			}
+			leaveFocus
+			{
+				setItemColor self backcolor 0 0 0 0;
+			}
+			action
+			{
+				close "self"; 
+                uiScript "voteTypeMap";
```
Comment (simonlfc) [inline iw4x/iw4x_00/ui_mp/scriptmenus/kickplayer.menu:121]: uiScript "voteKick";

File: iw4x/iw4x_00/ui_mp/scriptmenus/changegametype.menu (hunk 1 above)
Comment (diamante0018) [reply iw4x/iw4x_00/ui_mp/scriptmenus/changegametype.menu:121]: This one is `g_gametype`. But thanks for pointing out all the others, very helpful.

Comment (diamante0018) [reply iw4x/iw4x_00/ui_mp/scriptmenus/changegametype.menu:121]: My bad I had a moment over here

**Pull request merged**